## Service 
float32 goal_state_x
float32 goal_state_y
---
int8 FindPathFlag